contents/uix/accessibility.md

```diff
@@ -0,0 +1,181 @@
+<frontmatter>
+  title: Accessibility
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+{{ navbar | safe }}
+
+<div class="website-content">
+
+# Accessibility
+
+**Author(s): [Monika Manuela Hengki](https://github.com/monmanuela)**
+
+Reviewers: [Rahul Rajesh](https://github.com/rrtheonlyone), [Jacob Li Pengcheng](https://github.com/jacoblipech), [Tan Heng Yeow](https://github.com/tanhengyeow)
+
+## What is accessibility?
+
+> Accessibility is the practice of making your websites usable by as many people as possible — we traditionally think of this as being about people with disabilities, but really it also benefits other groups such as those using mobile devices, or those with slow network connections. <sub>--[MDN Web Docs](https://developer.mozilla.org/en-US/docs/Learn/Accessibility/What_is_accessibility)</sub>
+
+Note that although the scope of this article is the accessibility of websites, many of the points also apply to accessibility and UI/UX of other softwares too.
+
+## Why is accessibility important?
+
+Accessibility is important for users with disabilities to access the Web. There are many kinds of disabilities, including:
+   * Visual %%e.g., partial blindness, full blindness, color blindness, cataract, glaucoma, etc.%%
+   * Auditory %%e.g., hard of hearing, deafness, etc.%%
+   * Cognitive %%e.g., ADHD, autism spectrum, etc.%%
+   * Mobility %%e.g., quadriplegia, muscular dystrophy, etc.%%
+
+At the same time, accessibility also benefits people _without_ disabilities, for example:
+   * Device limitations %%e.g., using mobile phones, smart watches, smart TVs, and other devices with small screens, different input modes, etc.%%
+   * Age-related limitations %%e.g., poor dexterity, poor eyesight%%
+   * Temporary limitations %%e.g., broken arm, lost glasses%%
+   * Situational limitations %%e.g., under bright sunlight, an environment where one cannot listen to audio%%
+   * Internet-related limitations %%e.g., slow Internet connection, limited bandwidth%%
+
+Improving accessibility also brings some benefits to you as a developer:
+1. **Accessibility helps you improve your website's UX**. [W3C](https://www.w3.org/standards/webdesign/accessibility) notes that accessibility practices are good design practices in general, which improve the UX of your website.
+2. **Accessibility is a right**. Caring about accessibility demonstrates good ethics/morals.
```

Comment:
This point seems a bit different from the rest. Put it at the top of the list or as the last point?